shenyu-disruptor/src/main/java/org/apache/shenyu/disruptor/event/OrderlyDisruptorEventFactory.java

```diff
@@ -24,7 +24,7 @@
  * disruptor Create a factory implementation of the object.
  * @param <T> the type parameter
  */
-public class OrderlyDisruptorEventFactory<T> implements EventFactory<OrderlyDataEvent<T>> {
+public class OrderlyDisruptorEventFactory<T> implements EventFactory<DataEvent<T>> {
```

Comment:
why refactor OrderlyDataEvent to DataEvent

Reply:
the method DisruptorProviderManage#startup(boolean) has code:
```
        EventFactory<DataEvent<T>> eventFactory;
        if (isOrderly) {
            newConsumerSize = 1;
           // if OrderlyDisruptorEventFactory is OrderlyDataEvent ,it warning
            eventFactory = new OrderlyDisruptorEventFactory<>();
        } else {
            eventFactory = new DisruptorEventFactory<>();
        }
```

I've tried to clear up some of the generics issues that have been suppressed before, but haven't been successful. give me sometime